features/dual-stack-support.md

```diff
@@ -0,0 +1,238 @@
+
+[//]: # (The documentation instructions may exist on the developer wiki or the team drive. They may include instructions)
+[//]: # (for building running a `istioctl experimental command`, or using the preview profile,)
+[//]: # (or any other relevant information.)
+
+- [x] [Feedback plan](insert_your_link_here). N/A, we already have feedback from users that this is desired and useful.
+  There are some discussion on Dual Stack support as below issues:
+  
+  1. https://github.com/istio/istio/issues/24099
+  1. https://github.com/istio/istio/issues/32747
+  1. https://github.com/istio/istio/issues/34577
+  1. https://github.com/istio/istio/issues/35310
+  1. https://github.com/istio/istio/issues/35315
+  1. https://github.com/istio/istio/issues/40394
+  1. https://github.com/istio/istio/issues/41462
+
+[//]: # (This may include user feedback meetings, discuss.istio.io conversations, GitHub issues, or mailing lists.)
+
+- [x] Disabled by default.
+
+- [x] No impact on performance when the feature is disabled.
+
+
+[//]: # (Once all other items are completed, features.yaml should be updated to promote the feature)
+
+- [x] [features.yaml](https://github.com/istio/enhancements/blob/master/features.yaml) updated for this feature
+---
+
+## Alpha
+
+### Requirements: 
+
+**Design**
+
+- [ ] RFC has been approved describing the intention of the feature as well as the user stories behind the feature. 
+N/A
+
+**Config**
+
+- [x] Explicit user action is required to enable this feature (Please refer to the `Installation` section in [Supporting Dual Stack](https://docs.google.com/document/d/15LP2XHpQ71ODkjCVItGacPgzcn19fsVhyE7ruMGXDyU/edit?usp=sharing) ). 
+
+> Link to instructions for enabling
+
+**Docs**
+
+- [x] Reference docs are published to preliminary.istio.io or the Istio wiki.
+- [ ] Basic feature docs are published on preliminary.istio.io describing what the feature does, how to use it, and any caveats. 
+- [x] Release notes entries added as appropriate
+- [ ] Upgrade notes entries added as appropriate
+
+**Tests**
+
+- [x] Automated integration tests cover core use cases with the feature enabled. (The integration tests cover below use case: 1. Deployed Dual Stack k8s cluster, single stack Istio and Dual Stack services. 2. Based on item 1 to test the connectivity and accessibility, the same functionality verification as IPv6 support. More integration tests should need TODO.)
+- [X] When disabled, the feature does not affect system stability or performance. 
+
+**API**
+
+- [x] Initial API review.
+N/A
+
+**Approvals**
+
+- [x] The appropriate work group(s) have reviewed and approved promotion of the feature.
+- [x] The TOC has reviewed and approved promotion of the feature as part of the
```

Comment:
```suggestion
- [] The TOC has reviewed and approved promotion of the feature as part of the
```

This is "Alpha" but PR is experimental. Can you uncheck this?

same above and L150

Reply:
Hi @howardjohn, okay, unchecked